templates/form/fieldset.html.twig

```diff
@@ -36,5 +36,6 @@
 {% endif %}
 
 {% include '@components/fieldset/fieldset.twig' with {
-  modifier_classes: attributes.addClass(classes).removeClass(remove_classes).class|join(' ')|trim,
+  'attributes': attributes,
```

Comment:
Since we're passing the attribute classes in as modifier classes, I think the attributes here should be `attributes|without('class')` so we're not passing in the attribute classes twice. 

Same note applies to all templates where we're explicitly passing in attribute classes (with or without modifications) as modifier_classes.

Reply:
Good point. I refactored this a bit to make it explicit when we’re adding modifier classes, and making sure we don’t add duplicate classes or attributes.